gensrc/proto/segment.proto

```diff
@@ -146,6 +146,14 @@ message ZoneMapPB {
     optional bool has_not_null = 4;
 }
 
+// Metadata for JSON type column
+message JsonMetaPB {
+    // Format version
+    // Version 1: encode each JSON datum individually, as so called row-oriented format
+    // Version 2(WIP): columnar encoding for JSON
+    required uint32 format_version = 1;
```

Comment:
```suggestion
    optional uint32 format_version = 1;
```

Reply:
The `JsonMeta` is optional, and the `format_version` is required.

I think this field could not be optional, without which the network serialization procedure could not works.

Reply:
https://github.com/protocolbuffers/protobuf/issues/2497

I know your mean. but for protobuf, would better use optional even if the field is required currently.